libp2p/muxers/mplex/lpchannel.nim

```diff
@@ -247,6 +247,9 @@ method write*(s: LPChannel, msg: seq[byte]): Future[void] =
 
   s.completeWrite(fut, msg.len)
 
+method isCircuitRelay*(s: LPChannel): bool =
```

Comment:
nitpick, but I would name this `relayed`.

Reply:
Instead of this method, we could have a `getWrappedConnection`, which would return `s.conn` in this case. It would be more versatile

Reply:
I changed it to getWrapped and I used it [here](https://github.com/status-im/nim-libp2p/pull/717/files#diff-bd5997eecbb00b0d3a8a5121afba90ad5d7800037d398c5d0da2bd9663a53414R98-R104), isCircuitRelay/isRelayed could not really be re-used, getWrapped could eventually be.